include/mbedtls/ecp.h

```diff
@@ -53,139 +64,145 @@ extern "C" {
+ * \note For blinding, use odd in mbedtls_ecp_mul() and prime in
+ * mbedtls_ecdsa_sign().
  *
- * In the case of Montgomery curves, we don't store A but (A + 2) / 4 which is
- * the quantity actually used in the formulas. Also, nbits is not the size of N
- * but the required size for private keys.
+ * For Montgomery curves, we do not store \p A, but (A + 2) / 4, which is 
+ * the quantity used in the formulas. Additionally, \p nbits is not the 
+ * size of \p N but the required size for private keys.
  *
- * If modp is NULL, reduction modulo P is done using a generic algorithm.
- * Otherwise, it must point to a function that takes an mbedtls_mpi in the range
- * 0..2^(2*pbits)-1 and transforms it in-place in an integer of little more
- * than pbits, so that the integer may be efficiently brought in the 0..P-1
- * range by a few additions or substractions. It must return 0 on success and
- * non-zero on failure.
+ * If \p modp is NULL, reduction modulo \p P is done using a generic algorithm. 
+ * Otherwise, it must point to a function that takes an \p mbedtls_mpi in the 
+ * range of 0..2^(2*pbits)-1, and transforms it in-place in an integer of 
+ * little more than \p pbits, so that the integer may be efficiently brought 
+ * in the 0..P-1 range by a few additions or substractions.
+ *
+ * \return \c 0 on success
+ * \return Non-zero on failure.
  */
 typedef struct
 {
-    mbedtls_ecp_group_id id;    /*!<  internal group identifier                     */
-    mbedtls_mpi P;              /*!<  prime modulus of the base field               */
-    mbedtls_mpi A;              /*!<  1. A in the equation, or 2. (A + 2) / 4       */
-    mbedtls_mpi B;              /*!<  1. B in the equation, or 2. unused            */
-    mbedtls_ecp_point G;        /*!<  generator of the (sub)group used              */
-    mbedtls_mpi N;              /*!<  the order of G                                */
-    size_t pbits;       /*!<  number of bits in P                           */
-    size_t nbits;       /*!<  number of bits in 1. P, or 2. private keys    */
-    unsigned int h;     /*!<  internal: 1 if the constants are static       */
-    int (*modp)(mbedtls_mpi *); /*!<  function for fast reduction mod P             */
-    int (*t_pre)(mbedtls_ecp_point *, void *);  /*!< unused                         */
-    int (*t_post)(mbedtls_ecp_point *, void *); /*!< unused                         */
-    void *t_data;                       /*!< unused                         */
-    mbedtls_ecp_point *T;       /*!<  pre-computed points for ecp_mul_comb()        */
-    size_t T_size;      /*!<  number for pre-computed points                */
+    mbedtls_ecp_group_id id;    /*!< An internal group identifier. */
+    mbedtls_mpi P;              /*!< A prime modulus of the base field. */
+    mbedtls_mpi A;              /*!< \p A in the equation or <code>(A + 2) / 4</code>. */
```

Comment:
I think we should use `<code>` for all such formulas -- could you uniformize this throughout the file?

Reply:
done